fixtures/eurolite/led-tmh-14.json

```diff
@@ -0,0 +1,241 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED TMH-14",
+  "shortName": "TMH-14",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Victor Hampel"],
+    "createDate": "2022-06-09",
+    "lastModifyDate": "2022-06-09"
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.de/manual/286274/Eurolite-Led-Tmh-14.html"
+    ],
+    "productPage": [
+      "https://www.musikhaus-korn.de/en/eurolite-led-tmh-14-moving-head-zoom-wash/pd/134357"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=h45pZVr19tg"
+    ]
+  },
+  "physical": {
+    "dimensions": [145, 200, 280],
+    "weight": 3.25,
+    "power": 65,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "3× 12W COB RGBW LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [4, 32]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": ["Pan fine"],
+      "defaultValue": 32768,
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "540deg"
+      }
+    },
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "defaultValue": 32768,
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0deg",
+        "angleEnd": "190deg"
+      }
+    },
+    "Pan/Tilt Speed": {
+      "capability": {
+        "type": "PanTiltSpeed",
+        "speedStart": "fast",
+        "speedEnd": "slow"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [1, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "1Hz",
+          "speedEnd": "18Hz"
+        }
+      ]
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Zoom": {
+      "capability": {
+        "type": "Zoom",
+        "angleStart": "wide",
+        "angleEnd": "narrow"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Colors, color change": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 127],
+          "type": "ColorPreset",
+          "comment": "Static colors",
+          "helpWanted": "Which colors can be selected at which DMX values?"
+        },
```

Comment:
@Gamemap do you have access to the fixture? Would it be possible for you to find out the specific colors at the different DMX values?

Otherwise, let's just merge this as is, then the `helpWanted` message will be shown on the website.

Reply:
I can take a look at it tomorrow .

Reply:
Colors:

11-15 light blue
16-23 blue
24-31 purple
32-39 pink
40-47 rosa
48-71 red
72-79 orange
80-87 warm white (mixed from rgbw)
88-95 yellow
96-103 yellow green (mixed color german: gelbgün)
104-119 green
120-127 white (mixed rgbw)

Reply:
Thanks for looking up the colors!